submissions/expero1/oop-exercise/index.js

```diff
@@ -0,0 +1,123 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Habitat {
+  static SPECIES = { DOG: "dog", CAT: "cat", HUMAN: "human" };
+  static GENDER = { MALE: "male", FEMALE: "female" };
+  static LIMBS = { HANDS: "hands", LEGS: "legs" };
```

Comment:
Base class doesn't need to provide all sorts of limbs for all possible situations of child classes.
Why we cann't treat any limbs as any other properties like `species` or `name`?
If it is a shared property it is then defined at some shared (base) class. If a prop unique to a specific class so let that specific class define and handle that property.
Adding a class-specific property shouldn't affect any of its parents or children.
This what Open-Closed and encapsulation principles are.